components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -261,12 +261,12 @@ export default function () {
                 )}
             </div>
             <div>
-                <h3 className="mt-12">Delete Project</h3>
+                <h3 className="mt-12">Remove Project</h3>
```

Comment:
We are actually deleting the project here. Remove suggests that it is just somehow disconnected.

Reply:
cc @gtsiolis

Reply:
Ah! That ('remove') makes sense, thanks for clarifying 🙏